CONTRIBUTING.md

```diff
@@ -1,39 +1,38 @@
 # Contributing
 
-Thanks for your help! Due to ReScript's nature, the contribution setup isn't all straightforward. If something isn't working, please file an issue!
+Thanks for your help! Due to Melange's nature, the contribution setup isn't all straightforward. If something isn't working, please file an issue!
```

Comment:
this isn't correct. we use proper package management, so the setup should, in fact, be fairly straightforward.